In [3]:
from __future__ import print_function
import os
os.environ["PUIDATA"] = "%s/PUIDATA"%os.getenv("HOME")
import numpy as np
print ("PUIDATA", os.getenv("PUIDATA")) 

PUIDATA /nfshome/kyn227/PUIDATA


In [4]:
!curl https://raw.githubusercontent.com/fedhere/PUI2018_fb55/master/Lab4_fb55/times.txt > times.txt

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  1179  100  1179    0     0   7457      0 --:--:-- --:--:-- --:--:--  7462


In [5]:
!mv times.txt $PUIDATA

In [6]:
!ls $PUIDATA

201601-citibike-tripdata.csv  population_neighbourhood.csv
201601-citibike-tripdata.zip  times.txt
DHS.csv			      WaterConsumption.csv


In [7]:
timing = np.loadtxt("%s/times.txt"%os.getenv("PUIDATA"))

#### $H_{0}$: The commute time is on average the same or longer with the new bus route as it was before

#### $T_{new}$ >= $T_{old}$


#### $H_{1}$: The commute time is on average is shorter with the new bus route than it was before 

#### $T_{new}$ < $T_{old}$

In [8]:
sample_mean= timing.mean()

In [9]:
sample_std= timing.std()

In [10]:
population_mean=36
population_sd =6

In [11]:
z=(population_mean-sample_mean)/((population_sd/(100**0.5)))

In [12]:
print(z)

2.55639718617


The z-test shows a z-score of 2.56, which is > 2.
We reject $H_0$ at 0.05 significance level.

Conclusions about the null hypothesis: We reject the claim that the commuting time with the new bus route added, is the same or longer than before the new bus route was implemented. 